Yoast/YoastSEO.js
# 1854 improve example add assessor switching

## Summary

**Note**: this branch was branched from the #1826 branch. #1826 should be merged first before merging this branch into develop.

This PR can be summarized in the following changelog entry:

* Adds toggles to use the different assessors in the webpack example. E.g. for all the different combinations for cornerstone and taxonomy pages and related keyphrase.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Startup the webpack example.
   * I.o.w. go to `examples/webpack` in the console and enter the `yarn start` command.
* Test out all the different combinations of the `Is a related keyphrase`, `Is cornerstone article` and `is taxonomy page`.
   * For each combination, check whether all the assessments that are defined for this assessor are run (see [this wiki page](https://github.com/Yoast/YoastSEO.js/wiki/SEO-Assessors) for a list).
   * For each combination, (sanity) check whether all the assessments that are run provide the expected results.

Fixes #1854
